src/containers/RecipientsContainer.jsx

```diff
@@ -43,10 +44,24 @@ class RecipientsContainer extends Component {
     ) {
       this.toggleRemoveRecipientDialog()
     }
+
+    if (
+      (this.props.actionsPending.getRecipients || this.props.recipients) &&
+      !this.state.recipientsFetchStarted
+    ) {
+      // case 1. fetch recipients after mount
+      // use actionsPending to check fetch status
+      //
+      // case 2. accounts already fetched on other page
+      // use recipients prop to determine fetch status
+      this.setState({ recipientsFetchStarted: true })
```

Comment:
Why is this necessary? shouldn't be enough to just rely on this.props.actionsPending.getRecipients?

Reply:
gap between mount and actionsPending causing empty state being displayed as well as the add btn being enabled.

Reply:
We should consider moving getRecipient to postLoginPreparation.

Reply:
The gap happens when the component is mounted but getRecipient is not dispatched. I think this can happen when the app is overloading by getAccounts and syncWIthNetwork.

Reply:
will be another issue